cli/cli.go

```diff
@@ -330,12 +351,14 @@ var RootCmd = &cobra.Command{
 		// join args into a single string separated by spaces
 		prompt := strings.Join((args), " ")
 		apiKey := os.Getenv("OPENAI_API_KEY")
+		// The organization key is optional ( just for us enterprise users :) )
```

Comment:
Hahaha love it! If you git rid of "us" I'll keep the smiley ;D

Reply:
Okok just joking around in comments 😂